src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.Mono.cs

```diff
@@ -219,6 +213,34 @@ public sealed class AssemblyBuilder : Assembly
 
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         [DynamicDependency("RuntimeResolve", typeof(ModuleBuilder))]
+
+        [DynamicDependency("_mono_assembly")]
+        [DynamicDependency("_evidence")]
+        [DynamicDependency("dynamic_assembly")]
+        [DynamicDependency("entry_point")]
+        [DynamicDependency("modules")]
+        [DynamicDependency("name")]
+        [DynamicDependency("dir")]
+        [DynamicDependency("cattrs")]
+        [DynamicDependency("resources")]
+        [DynamicDependency("public_key")]
+        [DynamicDependency("version")]
+        [DynamicDependency("culture")]
+        [DynamicDependency("algid")]
+        [DynamicDependency("flags")]
+        [DynamicDependency("pekind")]
+        [DynamicDependency("delay_sign")]
+        [DynamicDependency("access")]
+        [DynamicDependency("loaded_modules")]
+        [DynamicDependency("win32_resources")]
+        [DynamicDependency("permissions_minimum")]
+        [DynamicDependency("permissions_optional")]
+        [DynamicDependency("permissions_refused")]
+        [DynamicDependency("peKind")]
+        [DynamicDependency("machine")]
+        [DynamicDependency("corlib_internal")]
+        [DynamicDependency("type_forwarders")]
+        [DynamicDependency("pktoken")]
```

Comment:
Nit: are you able to use nameof in any of these?

Reply:
Could you a comment as to why the dependencies are on this method.

Reply:
yeah, will switch to nameof